docs/conf.py

```diff
@@ -6,7 +6,7 @@
 
 from rocm_docs import ROCmDocs
 
-docs_core = ROCmDocs("ROCm Docs Core 0.0.2")
+docs_core = ROCmDocs("ROCm Docs Core", "0.4.0")
```

Comment:
should we not automate this as well?

Reply:
The pyproject.toml has multiple version strings with the same format. Since this is the only repo that uses that particular structure, it didn't seem worth adding another file with the version since updating that would be the same as updating it here.

Reply:
We automated the release including a bump of the version number via github action. If we don't automate this, it will go out of date ASAP. maybe leave it out for now.